modules/howtos/examples/analytics.py

```diff
@@ -87,9 +88,30 @@ def main(self, args):
 
         # tag::rowsasobject[]
         result = cluster.analytics_query(
-            "select * from `travel-sample` limit 10"
+            "select * from airports limit 10"
```

Comment:
Maybe we can add a comment to say what's required for this code snippet to work? Will you need an `airports` DATAVERSE (can't remember if it's still called that)?

Reply:
Yes! I've done that for the other SDK examples, will add...

(Though presumably the automation will add this too, at which point the dependencies will be specified in the README rather than needing to be dropped in various random files?)

Reply:
Yeah indeed! It might be useful for when we automate also to remind us what's required for the example to work. 👍